server/chat-commands/room-sections.ts

```diff
@@ -0,0 +1,94 @@
+import {Utils} from '../../lib';
+
+export type RoomSection = 'none' | 'nonpublic' | 'officialrooms' | 'officialtiers' | 'communityprojects' |
+'gaming' | 'languages' | 'entertainment' | 'lifehobbies' | 'onsitegames';
+
+export const sections: RoomSection[] = [
+	'none', 'nonpublic', 'officialrooms', 'officialtiers', 'communityprojects', 'gaming', 'languages', 'entertainment', 'lifehobbies', 'onsitegames',
+];
+
+export const sectionNames: {[k in RoomSection]: string} = {
+	none: 'none',
+	nonpublic: 'Non-public',
+	officialrooms: 'Official rooms',
+	officialtiers: 'Official tiers',
+	communityprojects: 'Community projects',
```

Comment:
Would probably be better to caps these. 
```suggestion
	communityprojects: 'Community Projects',
```

Reply:
i purposely didnt do it to match how ps currently does things, + i think all caps is ugly, especially next to hyphens